From: WinNative
Subject: [PATCH] tu/autotune: lower the GMEM bandwidth multiplier

WinNative's balanced tuning: weigh GMEM bandwidth at 10/10 instead of 11/10
when choosing between GMEM and sysmem rendering.
---
diff --git a/src/freedreno/vulkan/tu_autotune.cc b/src/freedreno/vulkan/tu_autotune.cc
index 5540c15..ec72154 100644
--- a/src/freedreno/vulkan/tu_autotune.cc
+++ b/src/freedreno/vulkan/tu_autotune.cc
@@ -1061,7 +1061,7 @@ struct tu_autotune::rp_history {
          /* Drawcalls access GMEM in GMEM rendering, but we do not want to ignore them completely.  The state changes
           * between tiles also have an overhead.  The magic numbers of 11 and 10 are randomly chosen.
           */
-         gmem_bandwidth = (gmem_bandwidth * 11 + total_draw_call_bandwidth) / 10;
+         gmem_bandwidth = (gmem_bandwidth * 10 + total_draw_call_bandwidth) / 10;
 
          bool select_sysmem = sysmem_bandwidth <= gmem_bandwidth;
          render_mode mode = select_sysmem ? render_mode::SYSMEM : render_mode::GMEM;
